xml/installation-installation-configure_lbaas.xml

```diff
@@ -180,22 +180,28 @@ openssl req -x509 -passin pass:foobar -new -nodes -key cakey.pem -out serverca_0
    <emphasis role="bold">Installing the Amphora Image</emphasis>
   </para>
   <para>
-   Octavia is utilizing &o_comp; VMs for its load balancing function and &vendor;
+   Octavia uses &o_comp; VMs for its load balancing function and &vendor;
    provides images used to boot those VMs called
-   <literal>octavia-amphora-haproxy</literal>.
+   <literal>octavia-amphora</literal>.
   </para>
   <warning>
    <para>
     Without these images the Octavia load balancer will not work.
    </para>
   </warning>
+  <para vendor="suse">
+   The &ostack; load balancing service (Octavia) does not automatically upload
+   the Amphora guest image. The image
+   <filename>/srv/www/suse-12.3/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image.rpm</filename>
+   should be placed on the &lcm; node and the image registered.
+  </para>
+  <itemizedlist vendor="hpe">
+   <listitem>
   <para>
    You can download the <emphasis role="bold">Octavia Amphora HA Proxy Guest
-   Image</emphasis> from &vendor; Software Entitlement Portal.
-   <!-- FIXME: is the naming of the portal correct going forward?
-   - sknorr, 2018-02-05 -->
+   Image</emphasis> from &hpe; Software Entitlement Portal.
```

Comment:
For this portal, we used to have the following URL (in other parts of the documentation): `http://www.hpe.com/software/entitlements`. However, this redirects to a Micro Focus page now.
Could you double-check that we really want this to be the "HPE Software Entitlements Portal"?